lib/registry.js

```diff
@@ -14,9 +13,62 @@ var BaseElement;
  */
 
 var ElementRegistry = createClass({
-  constructor: function() {
+  constructor: function(noDefault) {
     this.elementMap = {};
     this.elementDetection = [];
+    this.BaseElement = require('./primitives/base-element')(this);
```

Comment:
Something I think here, it would be really nice to be able to override/extend the base element. You have set it up here where I think that's possible, though you may want to keep that in mind when designing the interface for this registry. If so, you could in a sense add some method to all elements by changing the base element.